recipes/numpydoc

```diff
@@ -0,0 +1 @@
+(numpydoc :fetcher github :repo "douglasdavis/numpydoc.el" :branch "main")
```

Comment:
MELPA will pull your repository's default branch, so most likely the branch doesn't need to be specified in your recipe.  In fact, we prefer that you don't specify a branch, since this will make it easy for you to rename your default branch in the future without a followup PR.
```suggestion
(numpydoc :fetcher github :repo "douglasdavis/numpydoc.el")
```